loadbalancer/zk/client.go

```diff
@@ -0,0 +1,82 @@
+package zk
+
+import (
+	"github.com/go-kit/kit/log"
+	"github.com/samuel/go-zookeeper/zk"
+	"strings"
+	"time"
+)
+
+const ()
+
+// Client is a wrapper arround the zookeeper client.
+type Client interface {
+	// GetEntries queries the path in zookeeper, places a watch on it
+	// and retrieves data from its current child nodes
+	GetEntries(path string) ([]string, <-chan zk.Event, error)
+	// createParentNodes tries to create the path
+	// in case it does not exist yet on Zookeeper
+	createParentNodes(path string) error
+}
+
+type client struct {
+	*zk.Conn
+	evtConn <-chan zk.Event
+	acl     []zk.ACL
+	quit    chan struct{}
+}
+
+// NewClient returns a *zk.Client with a connection to the named server cluster.
+// It will return an error if the server cluster cannot be resolved.
+func NewClient(servers []string, acl *[]zk.ACL, logger log.Logger) (Client, error) {
+
```

Comment:
Please remove all extraneous newlines at the beginning of function implementation.